### PR TITLE
Add com.chmouel.gnomeNextMeetingApplet

### PR DESCRIPTION
**Pull request by chmouel**
Signed-off-by: Chmouel Boudjnah <chmouel@chmouel.com>

### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.chmouel.gnomeNextMeetingApplet.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+app-id: com.chmouel.gnomeNextMeetingApplet
+runtime-version: '41'
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+command: gnome-next-meeting-applet
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
```
Comment (hfiguiere) [inline com.chmouel.gnomeNextMeetingApplet.yaml:10]:
```suggestion
  - "--socket=fallback-x11"
  - "--socket=wayland"
```

File: com.chmouel.gnomeNextMeetingApplet.yaml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  - /share/girepository-1
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - /share/vala
+  - "*.la"
+  - "*.a"
+
+modules:
+  - shared-modules/intltool/intltool-0.51.json
```
Comment (hfiguiere) [inline com.chmouel.gnomeNextMeetingApplet.yaml:34]: you need to add the `shared-modules` git submodule for this to work.

File: com.chmouel.gnomeNextMeetingApplet.yaml [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        type: file
+        url: https://files.pythonhosted.org/packages/cc/72/48cfdd7a0caf3c27f392d2657731ac6f7c3c1c0a60bfeba3e1ba9ffa7ba9/PyGObject-3.42.0.tar.gz
+      - sha256: 961d03dc3453ebbc59dbdea9e4e11c5651520a876d0f4db161e8674aae935da9
+        type: file
+        url: https://files.pythonhosted.org/packages/36/7a/87837f39d0296e723bb9b62bbb257d0355c7f6128853c78955f57342a56d/python_dateutil-2.8.2-py2.py3-none-any.whl
+      - sha256: 8abb2f1d86890a2dfb989f9a77cfcfd3e47c2a354b01111771326f8aa26e0254
+        type: file
+        url: https://files.pythonhosted.org/packages/d9/5a/e7c31adbe875f2abbb91bd84cf2dc52d792b5a01506781dbcf25c91daf11/six-1.16.0-py2.py3-none-any.whl
+
+  - release.yaml
```
Comment (hfiguiere) [inline com.chmouel.gnomeNextMeetingApplet.yaml:125]: why is this a different file?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.chmouel.gnomeNextMeetingApplet.yaml (hunk 3 above)
Comment (chmouel) [reply com.chmouel.gnomeNextMeetingApplet.yaml:125]: this allows me to switch them to a dev.yaml which has dir when doing dev and [it's generated](https://github.com/chmouel/gnome-next-meeting-applet/blob/main/packaging/flatpak/update-release.sh) on release with the artefacts/sha, would you rather me to concat them?

File: com.chmouel.gnomeNextMeetingApplet.yaml (hunk 2 above)
Comment (hfiguiere) [reply com.chmouel.gnomeNextMeetingApplet.yaml:34]: given where the submodule is, it won't find this.

**Comment by hfiguiere** [comment]
bot, build com.chmouel.gnomeNextMeetingApplet

**Comment by chmouel** [comment]
ah ci error is : 

`stat: cannot stat 'builddir/*/share/app-info/xmls/com.chmouel.gnomeNextMeetingApplet.xml.gz': No such file or directory`

it seems that i didn't read the [documentation](https://www.freedesktop.org/software/appstream/docs/chap-CollectionData.html) properly on where to put the xml files

**Comment by hfiguiere** [comment]
it can't find the appdata file in `/app/share/{metainfo,appdata}/$FLATPAK_ID.{appdata,metainfo}.xml`

**Comment by hfiguiere** [comment]
not sure what's happening.

**Comment by chmouel** [comment]
it's because I forgot how to do "install(1)" :) 
(i needed to specify the target files to install not just a dir without a slash)

i can see from my local built it pick ups the file and rename it : 

```
Renaming com.chmouel.gnomeNextMeetingApplet.appdata.xml to share/appdata/com.chmouel.gnomeNextMeetingApplet.appdata.xml
```

**Comment by hfiguiere** [comment]
ah yeah install.

**Comment by hfiguiere** [comment]
I did look at the build steps and didn't catch it.

**Comment by hfiguiere** [comment]
bot, build com.chmouel.gnomeNextMeetingApplet

**Comment by chmouel** [comment]
I tested the CI build and it works ! 🥳 

image

**Pull request closed without merging**